eng/Packages.Data.props

```diff
@@ -47,8 +47,8 @@
     <PackageReference Update="System.Security.Cryptography.Algorithms" Version="4.3.0" />
     <PackageReference Update="System.Security.Cryptography.Cng" Version="4.3.0" />
     <PackageReference Update="System.Security.Cryptography.Primitives" Version="4.3.0" />
-    <PackageReference Update="System.Text.Encodings.Web" Version="4.5.1" />
-    <PackageReference Update="System.Text.Json" Version="4.7.2" />
+    <PackageReference Update="System.Text.Encodings.Web" Version="8.0.0" />
+    <PackageReference Update="System.Text.Json" Version="8.0.0" />
```

Comment:
We know that these are not safe to upgrade past the 6.0 line due to conflicts with Functions.  We also know that 6.0 is good with the currently supported versions of PowerShell.